expression/builtin_compare_vec.go

```diff
@@ -590,11 +592,78 @@ func (b *builtinNullEQStringSig) vecEvalInt(input *chunk.Chunk, result *chunk.Co
 }
 
 func (b *builtinLTIntSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinLTIntSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	var err error
+	if err = b.args[0].VecEvalInt(b.ctx, input, result); err != nil {
+		return err
+	}
+
+	n := input.NumRows()
+	buf, err := b.bufAllocator.get(types.ETInt, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf)
+	if err := b.args[1].VecEvalInt(b.ctx, input, buf); err != nil {
+		return err
+	}
+
+	return VecCompareInt(b.args[0], b.args[1], result, buf, resOfLT, result)
+}
+
+// VecCompareInt compares two integers.
+func VecCompareInt(lhsArg, rhsArg Expression, lColumn, rColumn *chunk.Column, f func(val int64, isNull bool, err error) (int64, bool, error), result *chunk.Column) error {
```

Comment:
It will be better to add a test for this function.

Reply:
ok, i will add a test for this function, thank you~